problems/combiner/problem.txt

```diff
@@ -1,5 +1,7 @@
-Write a module that returns a readable/writable stream using the
-`stream-combiner` module. You can use this code to start with:
+Create a new file named combiner.js, it should return a readable/writable stream using the
```

Comment:
I like the added instruction for making a file. Let's keep also the info that it's a module? It's more node-specific explanation which I think is good for learners at this stage.